tools/deny.toml

```diff
@@ -75,6 +82,10 @@ skip-tree = [
     # dependency tree because windows-sys has many sub-crates
     # that differ in major version.
     { name = "windows-sys", version = "=0.42.0" },
+
+    # TestSys uses a newer version of base64 and serde_yaml
+    { name = "testsys-model", version = "=0.0.7" },
+    { name = "bottlerocket-types", version = "=0.0.7" },
```

Comment:
Can we `skip-tree` the older versions rather than the newer ones? I find it more useful to consider what's holding us back from upgrading vs. tracking where we're already on new versions.

Reply:
There are several issues with TestSys 0.0.7 due to our change from openssl to rustls. This includes `base64`, `serde_yaml`, `thiserror`, `json_patch` and possibly others. Is it worth explicitly calling each one out?

Reply:
It seems fair to assume that testsys dependencies will be as current as possible, if we're following the same methodology there, so skipping the tree here makes sense to avoid duplication of effort.